### ISSUE TITLE
Feature: create admin/non-admin api-keys.

### ISSUE THREAD

**Issue by esune** [bcgov/traction#1028]
Currently, tenant users all have the same level of access and are allowed to perform any type of operation, even change configurations. It might be useful to be able to issue non-admin api-keys to consumers that only need to perform "basic" tasks, while users with admin access that can change tenant settings, etc. would have an admin api-key.

T.B.D. what exactly is considered an admin function and what is not.

**Comment by loneil** [comment]
Thinking about it, we should probably tie the ability to **create new keys** to admin keys only.

**Comment by esune** [comment]
> Thinking about it, we should probably tie the ability to **create new keys** to admin keys only.

Yes, definitely - I didn't explicitly state it, but that would be necessary.

**Comment by esune** [comment]
Commented there

**Comment by esune** [comment]
Closing as not planned, we'll tackle this functionality outside of Traction.
